frontend/src/types.ts

```diff
@@ -887,6 +893,11 @@ export enum DashboardItemMode {
     Edit = 'edit',
 }
 
+export interface DashboardAutoRefresh {
```

Comment:
This seems like a very specific type, perhaps should be added directly on `dashboardLogic`? Further, it's not actually used at all right now.